less/index.less

```diff
@@ -50,10 +50,165 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
+	
 }
 
 html, body {
     height: 100%;
 }
 
+// Variables
+@textcolor: rgba(47, 0, 255, 0.582);
+
+
+// Mixins
+.center() {
+	display: flex;
+	justify-content: center;
+	align-content: center;
+	
+}
+
+.font() {
+	font-size: 1.5rem;
+	font-family: Arial, Helvetica, sans-serif;
+	
+}
+
+
+// General Styles
+
+
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+	background-color: rgba(6, 34, 8, 0.562);
+
+  header {
+	display: flex;
+	justify-content: space-between;
+	margin: 30px;
+	align-items: center;
+	
+	h1 {
+		font-size: 3rem;
+		color: @textcolor;
+		@media(max-width: 500px ) {
+			.font();
+			}
+		}
+nav {
+	display: flex;
+	justify-content: space-around;
+	width: 50%;
+	@media(max-width: 500px ) {
+   width: 100%;
+			
+		}
+	a{
+		color: @textcolor;
+		.font();
+		@media(max-width: 500px ) {
+			font-size: 1rem;
+			}
+		
+	}
+}	
+	}
+  h2 {
+	font-size: 4rem;
+	margin: 20px 0;
+	color: @textcolor;
+	@media(max-width: 500px ){
+		font-size: 2.5rem;
+		color: @textcolor;
+		margin: 20px 0;
+}
+	
+	}
+
+
+	
+h3 {
+	.center();
+	font-size: 2rem;
+	padding: 20px 0;
+	color: @textcolor;
+  }
+  
+ .title {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	
+	a{
+	color: blue;
+	.font();
+	}
+	p{
+		margin: auto;
+		padding: 0;
+	}
+imgs {
+		display: flex;
+		justify-content: space-between;
+		padding: 20px;
+		@media(max-width: 800px ) {
+			display: flex;
+			flex-direction: column-reverse;
+			align-items: center;
```

Comment:
nice use of an extra breakpoint here to affect the layout in different viewsizes!